keyboards/kingly_keys/ave/ortho/keymaps/default/keymap.c

```diff
@@ -0,0 +1,229 @@
+ /* 
+ Copyright 2020 Garret Gartner
+  
+ This program is free software: you can redistribute it and/or modify 
+ it under the terms of the GNU General Public License as published by 
+ the Free Software Foundation, either version 2 of the License, or 
+ (at your option) any later version. 
+  
+ This program is distributed in the hope that it will be useful, 
+ but WITHOUT ANY WARRANTY; without even the implied warranty of 
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the 
+ GNU General Public License for more details. 
+  
+ You should have received a copy of the GNU General Public License 
+ along with this program.  If not, see <http://www.gnu.org/licenses/>. 
+ */ 
+
+#include QMK_KEYBOARD_H
+
+enum avenue_layers {
+  _QWERTY,
+  _LOWER,
+  _RAISE,
+  _FUNCTION
+};
+
+
+
+//Tap Dance Declarations:
+enum avenue_tapcodes {
+  TD_RST = 0,
+  TD_DBQT = 1
```

Comment:
```suggestion
  TD_RST,
  TD_DBQT,
```

You can omit the explicit value assignment here as `enum` starts at zero.